rfc8446.xml

```diff
@@ -7437,6 +7455,26 @@ Nikos. Anyway, this is an IESG matter and you are AD, so perhaps you can
 resolve it or talk to Kathleen. -->
 
 </section>
+
+<section anchor="sharing-psks" title="Sharing PSKs">
+<t>
+TLS 1.3 takes a conservative approach to PSKs by binding them to a
+specific KDF. By contrast, TLS 1.2 allows PSKs to be used with any
+hash function and the TLS 1.2 PRF. Thus, any PSK which is used with
+both TLS 1.2 and TLS 1.3 must be used with only one hash in TLS 1.3,
+which is less than optimal if users want to provision a single PSK.
+In addition, while constructions in TLS 1.2 and TLS 1.3, although both
+based on HMAC, are very different and there is no known way in which
+reuse of the same PSK in TLS 1.3 and TLS 1.2 would produce related
+output, only limited analysis has been done of the safety of this
+practice. Implementations can ensure safety from cross-protocol key
```

Comment:
This sentence is pretty long (and the "while" and "only limited analysis" clauses are pretty far removed).